server.js

```diff
@@ -4,23 +4,20 @@ const express = require('express');
 const sa = require('superagent');
```

Comment:
I believe you need either superagent or express-request-proxy.  You are no longer using superagent, so you can get rid of it.